client-runtime/src/main/java/com/microsoft/rest/v2/RestProxy.java

```diff
@@ -560,7 +401,7 @@ else if (returnTypeToken.isSubtypeOf(Single.class)) {
                 public Single<?> apply(HttpResponse response) throws Exception {
                     return handleRestResponseReturnTypeAsync(response, methodParser, singleTypeParam);
                 }
-            });
+            }).onErrorResumeNext(WARN_MISSING_DECODING);
```

Comment:
This slightly strange way of adding the missing decoder warning is due to the fact that certain methods of composition don't work with wildcards.